clients/imodels-client-management/src/base/internal/UtilityFunctions.ts

```diff
@@ -27,3 +27,11 @@ export async function waitForCondition(params: {
   const timeoutError = params.timeoutErrorFactory();
   throw timeoutError;
 }
+
+export function isTransientHttpStatusCode(statusCode: number) {
+  // TODO: we could handle 429, but then we need to take into account retry-after header value
+  return (
```

Comment:
I am not sure we ever return 408 either. We could simplify this and have retries for status codes >= 500. If you feel like there's no time to handle 429 properly, mark that information in the work item that we need to come back to this

Reply:
Removed 408, to handle 429 we'll have to get headers from iModels error, which currently does not contain them